resources/aws/auto_scaling_group_test.go

```diff
@@ -0,0 +1,54 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+)
+
+func TestIsLegacyASG(t *testing.T) {
+	nameOk := "my-asg"
+	clusterOk := "my-cluster"
+	tagNameOK := tagKeyName
+	tagClusterOK := tagKeyCluster
+	tagCF := "aws:cloudformation:stack-id"
+	nameBad := "my-other-asg"
+	clusterBad := "my-other-cluster"
+
+	testCases := []struct {
+		description string
+		name        string
+		clusterID   string
+		tags        []*autoscaling.TagDescription
+		expected    bool
+	}{
+		{"name and cluster match, no CF tag", nameOk, clusterOk, []*autoscaling.TagDescription{
+			{Key: &tagNameOK, Value: &nameOk},
+			{Key: &tagClusterOK, Value: &clusterOk},
+		}, true},
```

Comment:
I know no tool enforces this but my personal preference would be the explicit way. Not using the property name when initializing a structure is not really readbale. 

```
{
  description: ...,
  name: ...,
  clusterID: ...,
  tags: ...,
  expected: ...,
},
```

Reply:
Indeed, added, much better :)